kuma/static/js/wiki.js

```diff
@@ -976,10 +976,40 @@
         initDetailsTags();
     }
 
-    // clear out any drafts from localStorage, now that the document has been saved
-    if (typeof document_saved !== 'undefined' && document_saved) {
-      localStorage.removeItem('draft/edit' + location.pathname);
-      localStorage.removeItem('draft/edit' + location.pathname + '#save-time');
+
+    //
+    // Generates a storage key based on pathname
+    // copied from wiki-edit-draft.js because: race conditions
+    // does not need to copy logic for dealing with new translations
+    //
```

Comment:
MDN is the wild west of javascript formatting right now (I just copy what is in the file I'm working on at the time). Cross project standards would be awesome. 👍

I will try to get over my distaste for using comments that are not left aligned :P